app/models/concerns/article.rb

```diff
@@ -13,7 +13,7 @@ module Article
     before_validation :set_user
     validates_presence_of :user
 
-    after_commit :update_elasticsearch_data, :send_to_alegre, :notify_bots, on: [:create, :update]
+    after_commit :update_elasticsearch_data, :notify_bots, on: [:create, :update] #:send_to_alegre, 
```

Comment:
I thought the issue was only with feed/request?

Reply:
It is apparently not - how would you like to proceed now knowing this?

Reply:
This is a different `send_to_alegre`. It is simply a stub in `explainer.rb` at present. We do not need to comment and remove here.